scripts/apple_compile.sh

```diff
@@ -24,9 +24,12 @@ helpFunction() {
   print N "By default, the project is compiled in release mode. Use -d or --debug for a debug build."
   print N "Use -n or --networkextension to force the network-extension component for MacOS too."
   print N ""
+  print N "If MVPN_ADJUST_TOKEN env is found, this will be used at compilation time."
```

Comment:
Can we maybe call it `MVPN_ADJUST_TOKEN_IOS` to make clear that it is the iOS token?